tools/testing-server/plugins/agent-injector/loader-transform.js

```diff
@@ -20,13 +20,14 @@ async function getLoaderContent (request, reply, testServer) {
       testServer.config.polyfills ? '-polyfills' : ''
     }.min.js`
   )
-  const loaderFileStats = await fs.promises.stat(loaderFilePath)
 
-  if (!loaderFileStats.isFile()) {
+  let file
+  try {
+    file = (await fs.promises.readFile(loaderFilePath)).toString()
+  } catch (_) {
     throw new Error(`Could not find loader file ${loaderFilePath}`)
   }
-
-  return (await fs.promises.readFile(loaderFilePath)).toString()
+  return file
```

Comment:
Is this to keep the server from crashing if the requested loader file is not found?

Reply:
Yes, I tweaked this a bit to show an actual full error message during a troubleshooting session because it actually threw an unhandled exception before on bad cases.